src/tests/Interop/COM/ComWrappers/API/ComWrappersTestsBuiltInComDisabled.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Program.cs" />
```

Comment:
Isn't this just going to fail all the tests? I don't understand what we are trying to do here for validation.

Reply:
What are you expecting to fail? `ComWrappers` should still work properly with built-in COM support disabled, so the `ComWrappers/API` tests should work?

Reply:
Oops. I misunderstood this. Sorry.

Reply:
This test is the same as the default API project but with disabling built-in COM using the RuntimeHostConfigurationOption support you added recently to the corerun infrastructure.